great-idea/css/index.css

```diff
@@ -82,11 +94,21 @@ body{
 	flex-wrap: wrap; */
 }
 
-.content{
-	background-color: lightgray;
-	/* width: 100%; */
-	border: 1px solid black;
-	margin: 20px 0 0px 0;
+.content-right{
```

Comment:
I like how you  grouped this content together and the styling looks good on-screen.